src/parser/jobs/war/index.tsx

```diff
@@ -16,12 +16,12 @@ export const WARRIOR = new Meta({
 	</>,
 
 	supportedPatches: {
-		from: '✖',
-		to: '✖',
+		from: '7.0',
+		to: '7.01',
 	},
 
 	contributors: [
-	// {user: CONTRIBUTORS.YOU, role: ROLES.YOUR_ROLE},
+		{user: CONTRIBUTORS.KWEREY, role: ROLES.DEVELOPER},
```

Comment:
I might be wrong here (and another old hand is welcome to correct me), but I _think_ we prefer to have someone flagged as the maintainer role when a job is officially supported as the main point of contact for any issues that come up. If you're willing, it'd be better to set `ROLES.MAINTAINER` here.

Reply:
Let me give the caveat that I don't actually know much at all about Typescript, so my ability to fix any complicated problem is pretty limited! But I've taken enough of a look around the warrior modules to understand what they currently do and pick off some low hanging fruit issues, happy to mark myself as maintainer if it'd be in a state of neglect otherwise.